<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="horizontalMenu" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="refreshDataButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Images/res/images/sync.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addToFavButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Images/res/images/add.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeFavButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Images/res/images/remove.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="genGraphButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Images/res/images/chart.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="liveTrackButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Images/res/images/chart_descend.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_mainTable">
           <attribute name="title">
            <string>Stock Data</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableView" name="tableView">
              <property name="autoScrollMargin">
               <number>18</number>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_charts">
           <attribute name="title">
            <string>History Chart</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QWidget" name="graphWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QWidget" name="graphDateWidget" native="true">
                   <property name="maximumSize">
                    <size>
                     <width>550</width>
                     <height>30</height>
                    </size>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="spacing">
                     <number>11</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="maximumSize">
                       <size>
                        <width>30</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>From</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDateEdit" name="dateEdit">
                      <property name="maximumSize">
                       <size>
                        <width>120</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="displayFormat">
                       <string>d/M/yy</string>
                      </property>
                      <property name="calendarPopup">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="maximumSize">
                       <size>
                        <width>20</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>To</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDateEdit" name="dateEdit_2">
                      <property name="maximumSize">
                       <size>
                        <width>120</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="displayFormat">
                       <string>d/M/yy</string>
                      </property>
                      <property name="calendarPopup">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="generateGraphByDate">
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Generate</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="minMaxCheckbox">
                      <property name="text">
                       <string>Show min/max on graph</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCustomPlot" name="graph1" native="true"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QProgressBar" name="progressBar">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionItem_1"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionItem_1">
   <property name="text">
    <string>Item 1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>lib/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
